core.osgi/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/osgi/DefaultRepairConfigurationProviderComponent.java

```diff
@@ -61,13 +62,19 @@ public synchronized void activate(Configuration configuration)
         if (configuration.enabled())
         {
             long repairInterval = configuration.repairIntervalSeconds();
+            long repairSizeInBytes = Long.MAX_VALUE;
```

Comment:
Nit: `private static final long DEFAULT_TARGET_REPAIR_SIZE_IN_BYTES = Long.MAX_VALUE;`

Reply:
Same as in RepairProperties, referencing RepairConfiguration.FULL_REPAIR_SIZE